### PR TITLE
Adding Master Yi

### PR DESCRIPTION
**Pull request by lipeph99**
Create Master Yi folder, Add Highlander skill and buff

### CODE REVIEW

File: Champions/MasterYi/R.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+using LeagueSandbox.GameServer.API;
```
Comment (MythicManiac) [inline Champions/MasterYi/R.cs:1]: What's going on with the indentation in this file? Half seem to use tabs and the other half spaces, please make them consistent.

File: Buffs/Highlander/Highlander.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
+using LeagueSandbox.GameServer.GameObjects.AttackableUnits.AI;
```
Comment (MythicManiac) [inline Buffs/Highlander/Highlander.cs:1]: All wonky indentation in this file as well. If you use Visual Studio as is recommended, you can make the IDE autoformat the entire file on save (or just trigger it manually).

**Review by MythicManiac — COMMENTED** [review]
Looks good otherwise, but I'd like you to fix the indentation before I merge.

Nice work 👍

File: Buffs/Highlander/Highlander.cs (hunk 2 above)
Comment (lipeph99) [reply Buffs/Highlander/Highlander.cs:1]: Copied some parts and edited everything via github lol
Going to fix it

**Comment by MythicManiac** [comment]
This is better, although I think we have our other files using space as indention. That's the default for Visual Studio and also displayed by github better (GitHub has tabs as 6 characters wide).

So if you could change them to indentations of 4 spaces per level, that'd be much appreciated, sorry for not clarifying earlier 😄.

**Comment by lipeph99** [comment]
Sorry, I'm new on GitHub and C# :laughing:
I created a new Pull request with the 4 space indentations as requested

**Comment by MythicManiac** [comment]
No worries, are you on our Discord yet? If not, feel free to hop on (follow the link on the readme) and ask questions.

**Comment by MythicManiac** [comment]
Closing this PR in favor of #71

**Pull request closed without merging**